src/main/resources/data/legal-entities-with-users.json

```diff
@@ -134,5 +134,67 @@
         "role": "employee"
       }
     ]
+  },
+  {
+    "legalEntityName": "Boats of Atlantis",
+    "legalEntityExternalId": "boats_of_atlantis",
```

Comment:
Maybe better to use a more practical externalIds like B000001, B000002 etc.

Reply:
Now all users are U0001016-U0001022 and legal entities are B000001-B000005